packages/react-core/src/components/NumberInput/examples/NumberCustomStep.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { NumberInput } from '@patternfly/react-core';
+
+export const CustomStepNumberInput: React.FunctionComponent = () => {
+  const [value, setValue] = React.useState(90);
+  const step = 3;
+
+  const stepper = (stepValue: number) => {
+    setValue(value + stepValue);
+  };
+
+  const onChange = event => {
```

Comment:
Could you add the type to the event prop in all the onChange handlers?

Reply:
Because of this message: `Property 'value' does not exist on type 'EventTarget'.`, I've made the type assertion: `const target = event.target as HTMLInputElement;` With that, I think everything should be working and the event prop have its type.